docs/docs/tutorials/classification_finetuning/index.ipynb

```diff
@@ -35,7 +35,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```

Comment:
There are a lot of changes to this file but I'm not sure if they're actually intentional